src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookup.class.php

```diff
@@ -563,8 +563,11 @@ public function toString()
         if (!$this->data) {
             return '';
         }
+
         $tableName = $this->GetConnectedTableName();
-        $identifyingColumnName = $this->oDefinition->_oTableConf->GetNameColumn();
+        $connectedTableConf = TdbCmsTblConf::GetNewInstance();
+        $connectedTableConf->LoadFromField('name', $tableName);
```

Comment:
you should check if it was possible to load the table (`if ($connectedTableConf->LoadFromField('name', $tableName) {...`)

Reply:
ok